packages/components/src/components/icon/mt5/ic-mt5-financial-bg.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No need to run `svgo`? (same for other SVGs)

Reply:
Run SVGO on all the newly added svg files.